src/types.rs

```diff
@@ -821,6 +821,8 @@ pub enum SegmentType {
     GNU_STACK = 0x6474e551,
     /// Read-only after relocation
     GNU_RELRO = 0x6474e552,
+    /// Indicates whether an executable stack is needed
+    PT_GNU_STACK = 0x65041580,
```

Comment:
we remove the PT_ prefixes because rust isnt a global namespace. if you do that you see there's a conflicting type  above. I don't think 0x65041580 is GNU_STACK.

Reply:
looks like 0x65041580 is PT_PAX_FLAGS. yet another incompatible variant of memory protection.